packages/ui/src/views/repo/pull-request/pull-request-item-description.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC } from 'react'
+
+import { Icon } from '@/components'
+
+interface PullRequestItemDescriptionProps {
+  number: number
+  reviewRequired?: boolean
+  tasks?: number
+  author: string
+  sourceBranch: string
+  timestamp: string
+}
+
+export const PullRequestItemDescription: FC<PullRequestItemDescriptionProps> = ({
+  reviewRequired,
+  number,
+  tasks,
+  author,
+  sourceBranch,
+  timestamp
+}) => {
+  return (
+    <div className="text-14 text-foreground-4 inline-flex max-w-full items-center gap-1.5 pl-[22px] leading-none">
+      <p>
+        {!!number && `#${number}`} opened {timestamp} by {author}
```

Comment:
```suggestion
        {`#${number}`} opened {timestamp} by {author}
```

The `number` parameter is marked as required in the types, so we shouldn’t be checking for its existence. If this value can be omitted, we should mark it as an optional parameter in the types.

```tsx
interface PullRequestItemDescriptionProps {
  number: number
 ...
}
```